modules/delegates/api/fetchDelegates.ts

```diff
@@ -302,3 +313,139 @@ export async function fetchDelegates(
 
   return delegatesResponse;
 }
+
+export async function fetchDelegatesPaginated(
+  queryParams: DelegatesAPIQueryParams
```

Comment:
do not send query params to the method, send "start" "limit" or "page" "pageSize" parameters

Query params are related to the API endpoint. This is a function that should not know about them

Reply:
I should change the name of that function argument, it's not really the raw query params that the API endpoint is receiving, but the validated, filtered list of them, so I'm just passing to that function the params I need to return the right list of delegates